skills/pipeline/scripts/workspace-preflight.mjs

```
@@ -0,0 +1,129 @@
+    directory = await mkdtemp(join(root, PROBE_PREFIX));
+    file = join(directory, "probe");
+    handle = await open(
+      file,
+      fsConstants.O_CREAT | fsConstants.O_EXCL | fsConstants.O_WRONLY | fsConstants.O_NOFOLLOW,
+      0o600,
+    );
+    await handle.writeFile("team-harness-workspace-preflight\n", "utf8");
+  } finally {
+    if (handle) await handle.close();
+    if (file) {
+      try { await unlink(file); } catch (error) { if (error?.code !== "ENOENT") throw error; }
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)